src/state/machines/turn.ts

```diff
@@ -332,21 +313,24 @@ export const TurnMachine = setup({
         }
 
         draft.policyMarket.deck = without(context.policyMarket.deck, policyCard);
+        const cardEffect = policyCard.effect;
+        const hasFunding = TurnMachineGuards.hasSufficientFunding({ context });
+        const isMeasureCard = cardEffect === "positive";
+        const isHabitatAutoDraw = draft.turn.automaticPolicyDraw?.cause === "habitat";
+        const isExtinctionAutoDraw = draft.turn.automaticPolicyDraw?.cause === "extinction";
+
+        const base = "policy_policy";
+        const cause = isHabitatAutoDraw ? "AutoDrawHabitat" : isExtinctionAutoDraw ? "AutoDrawExtinction" : "Draw";
+        const effect = isMeasureCard
+          ? (`Positive${hasFunding ? "HasFunding" : "NoFunding"}` as const)
+          : capitalize(cardEffect);
+        const eventType = `${base}${cause}${effect}` as const;
+
+        const causeUids = isHabitatAutoDraw ? draft.turn.unlockedHabitats : undefined;
```

Comment:
If we are doing this for habitats, we should probably do the same for extinctions?

Reply:
Yeah I thought to do it separately later. IMO it's not so important since ext. tiles are all the same.

Reply:
Ok cool lets leave it for now, but please add issue so we dont forget👍🏻